packages/terra-date-picker/package.json

```diff
@@ -40,9 +40,11 @@
     "terra-responsive-element": "^4.0.0"
   },
   "devDependencies": {
+    "terra-disclosure-manager": "^4.9.0",
```

Comment:
It doesn't look like this, `terra-modal-manager`, or `terra-dialog` are used anywhere.

Reply:
Oops, good catch, that was leftover from testing. Removed now.